server/service/src/invoice_line/stock_out_line/mod.rs

```diff
@@ -1,2 +1,22 @@
 pub mod insert;
+use repository::InvoiceRowType;
+
 pub use self::insert::*;
+
+pub mod update;
+pub use self::update::*;
+
+#[derive(Clone, Debug, PartialEq)]
+pub enum StockOutType {
+    OutboundShipment,
+    Prescription,
+}
+
+impl StockOutType {
```

Comment:
Nice, I like this common type, although it did introduce some changes in `inserts`.

In the PR description you have:

>  plus some refactoring.

I think it would be great if you could have mentionted this refactor, would have helped me in the review (as in when I noticed changes in inserts, i was at first confused). So a comment like below would have been quite usefull in PR description:

"Refactored InsertOutType to StockOutType since it's a common type now with Update"

Reply:
Comment up top xD